packages/zoe/test/unitTests/test-zoe.js

```diff
@@ -66,6 +66,10 @@ test(`zoe.startInstance no issuerKeywordRecord, no terms`, async t => {
   t.deepEqual(result.creatorInvitation, undefined);
   t.deepEqual(result.instance, result.instance);
   t.deepEqual(result.publicFacet, {});
+  t.deepEqual(Object.getOwnPropertyNames(result.adminFacet), [
```

Comment:
```suggestion
  t.deepEqual(Object.getOwnPropertyNames(result.adminFacet).sort(), [
```

The test should not be sensitive to the order of names produced by `getOwnPropertyNames`. Similar issue on line 58. In both places, why are we checking an exact match on property names?

Reply:
I can't tell if this is a rhetorical question covered by your suggested fix, so I'll respond. If it was intended as rhetorical please ignore what follows.

We certainly have other tests that make use of each of the properties, but in this test, we're just testing the basic connectivity. Even though the contract used here doesn't do anything substantive, startInstance() is still supposed to return each of these fields, even if in a trivial state.